src/particleContainer/LinkedCells.cpp

```diff
@@ -1057,18 +1064,20 @@ void LinkedCells::updateBoundaryAndHaloMoleculeCaches() {
 	#if defined(_OPENMP)
 	#pragma omp parallel for schedule(static)
 	#endif
-	for (long int cellIndex = 0; cellIndex < (long int) _cells.size(); cellIndex++) {
+	for (size_t cellIndex = 0; cellIndex < _cells.size(); cellIndex++) {
 		if (_cells[cellIndex].isHaloCell() or _cells[cellIndex].isBoundaryCell()) {
 			_cells[cellIndex].buildSoACaches();
 		}
 	}
 }
 
 void LinkedCells::updateMoleculeCaches() {
+	const int chunk_size = chunk_size::getChunkSize(_cells.size(), 10000, 100);
```

Comment:
Either give the reasoning for these numbers or at least say that these are empirically optimized magic numbers. This comment applies to all calls of this function.